src/dosarray_configure_networking.sh

```diff
@@ -79,4 +79,4 @@ echo "Running CMD=${CMD}"
 echo
 echo "Please enter sudo password for ${TARGET_PHYSICAL_HOST}"
 
-dosarray_execute_on "${TARGET_PHYSICAL_HOST}" "-t ${CMD}"
```

Comment:
This change (removing the "-t") doesn't seem to be related specifically to this commit, judging by the commit message.

Reply:
-t parameter in the network config script interferes with the generation of commands in case of nerds. So I moved it within disarray_execute_on() in config.sh